src/patternfly/components/DataList/data-list.scss

```diff
@@ -67,6 +67,14 @@
   --pf-c-data-list__cell--cell--PaddingTop: 0;
   --pf-c-data-list__cell--cell--md--PaddingTop: var(--pf-global--spacer--lg);
   --pf-c-data-list__cell--m-icon--cell--PaddingTop: var(--pf-global--spacer--lg);
+  --pf-c-data-list__cell--MaxWidth: none;
+  --pf-c-data-list__cell--MinWidth: 0;
```

Comment:
```suggestion
  --pf-c-data-list__cell--MinWidth: initial;
```

Reply:
I would leave this as `auto` since it's the default, just to keep the default behavior. Then update `--m-truncate--MinWidth` to `5ch`